data_structures/hashing/hash_table.py

```diff
@@ -7,13 +7,18 @@ class HashTable:
     Basic Hash Table example with open addressing and linear probing
     """
 
-    def __init__(self, size_table, charge_factor=None, lim_charge=None):
+    def __init__(
+        self,
+        size_table: int,
+        charge_factor: int | None = None,
+        lim_charge: float | None = None,
```

Comment:
Python and mypy can figure this out and it is a suggestion to the caller to pass in an int and not pass in `None`.
```suggestion
        charge_factor: int = None,
        lim_charge: float = None,
```

Reply:
if you have a strong opinion, i can change - but personally this looks more legible is all. :)